src/mobile/components/Footer/index.tsx

```diff
@@ -0,0 +1,45 @@
+import classnames from 'classnames';
+import * as React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { HomeIcon } from '../../assets/images/footer/HomeIcon';
+import { OrderIcon } from '../../assets/images/footer/OrderIcon';
+import { ProfileIcon } from '../../assets/images/footer/ProfileIcon';
+import { TradeIcon } from '../../assets/images/footer/TradeIcon';
+import { WalletIcon } from '../../assets/images/footer/WalletIcon';
+
+const handleGetActiveItemClass = (currentRoute: string, targetRoute: string, absolute?: boolean) => {
+    return classnames('pg-mobile-footer__item', {
+        'pg-mobile-footer__item--active': absolute ? currentRoute === targetRoute : currentRoute.includes(targetRoute),
+    });
+};
+
+const FooterComponent = () => {
+    const currentRoute =  useLocation().pathname;
+
+    return (
+        <div className="pg-mobile-footer">
+            <Link to="/" className={handleGetActiveItemClass(currentRoute, '/', true)}>
```

Comment:
Maybe it is better to create an additional component
`['orders', 'trading', ...].map((url) => <FotterItem url={url}/>`